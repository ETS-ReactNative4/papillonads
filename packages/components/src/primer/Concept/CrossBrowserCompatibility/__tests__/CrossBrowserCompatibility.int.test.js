import {
  specificationsStandardsGuidelines,
  openSourceDependencyProjects,
  designSystemCore,
  productMicrosites,
} from './CrossBrowserCompatibility.int.story'

describe('<CrossBrowserCompatibility />', () => {
  describe('Render', () => {
    test('must match specificationsStandardsGuidelines()', () => {
      expect(global.renderToJSON(specificationsStandardsGuidelines())).toMatchSnapshot()
    })

    test('must match openSourceDependencyProjects()', () => {
      expect(global.renderToJSON(openSourceDependencyProjects())).toMatchSnapshot()
    })

    test('must match designSystemCore()', () => {
      expect(global.renderToJSON(designSystemCore())).toMatchSnapshot()
    })

    test('must match productMicrosites()', () => {
      expect(global.renderToJSON(productMicrosites())).toMatchSnapshot()
    })
  })
})
