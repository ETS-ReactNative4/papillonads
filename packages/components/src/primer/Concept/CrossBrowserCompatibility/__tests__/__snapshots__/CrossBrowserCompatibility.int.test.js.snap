// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CrossBrowserCompatibility /> Render must match renderDesignSystemCore() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Papillon Ads Implementation
  </h2>
  <h3>
    📖. Design System Core
  </h3>
</div>
`;

exports[`<CrossBrowserCompatibility /> Render must match renderOpenSourceDependencyProjects() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Open Source Dependency Projects
  </h2>
</div>
`;

exports[`<CrossBrowserCompatibility /> Render must match renderProductMicrosites() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Papillon Ads Implementation
  </h2>
  <h3>
    📖. Product Microsites
  </h3>
</div>
`;

exports[`<CrossBrowserCompatibility /> Render must match renderSpecificationsStandardsGuidelines() 1`] = `
<div
  className={null}
>
  <h2>
    📗 Specifications/Standards/Guidelines
  </h2>
  <h3>
    📖. ES6 / ES2015
  </h3>
  <h4>
    📃. From var to const/let
  </h4>
  <h5>
    📄. Reference
  </h5>
  <h6>
    📝. MDN
  </h6>
  <div>
    ﹘ 
    <a
      href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const"
    >
      const
    </a>
     Constants are block-scoped, much like variables defined using the let keyword. The value of a constant can't be changed through reassignment, and it can't be redeclared.
  </div>
  <div>
    ﹘ 
    <a
      href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let"
    >
      let
    </a>
     The let statement declares a block scope local variable, optionally initializing it to a value.
  </div>
  <h5>
    📄. Compatibility
  </h5>
  <h6>
    📝. Browser Support
  </h6>
  <div>
    ﹘ const &gt; Chrome ✓, Firefox ✓, Safari ✓, Edge ✓, Internet Explorer ✓
  </div>
  <div>
    ﹘ let &gt; Chrome ✓, Firefox ✓, Safari ✓, Edge ✓, Internet Explorer 🌦
  </div>
  <h6>
    📝. 🔖 Transpiled / Polyfilled
  </h6>
  <div>
    ﹘ Transpiled ✓, Safe to use ✅
  </div>
</div>
`;
